.github/workflows/pr-assign.yml

```
@@ -0,0 +1,19 @@
+name: Auto Request Review
+
+on:
+  pull_request:
+    # TODO remove synchronize event
+    types: [opened, ready_for_review, reopened, synchronize]
+
+jobs:
+  auto-request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
```

Minor, can we add a version comment that states what semver version this commit is tied to?
```suggestion
      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # 2.4.0
```